modules/Bio/EnsEMBL/IO/Parser/VCF4.pm

```diff
@@ -44,13 +44,13 @@ use base qw/Bio::EnsEMBL::IO::Parser::BaseVCF4/;
 sub open {
     my ($caller, $filename, @other_args) = @_;
     my $class = ref($caller) || $caller;
-     
```

Comment:
This is not valid. VCF supports *only* tab as its column separator.

https://samtools.github.io/hts-specs/VCFv4.2.pdf

Spaces can appear within fields (though obviously this might be discouraged, it is not explicitly disallowed).